srcDoc/custom_types.rst

```diff
@@ -0,0 +1,578 @@
+            Reliability: Reliable
+            Keyed: No
+            Subscriber ID: 0
+            Receive using: Listeners
+            Domain: 1
+            Dynamic Data: No
+            XML File: perftest_qos_profiles.xml
+
+        Transport Configuration:
+            Kind: UDPv4 & SHMEM (taken from QoS XML file)
+            Use Multicast: False
+
+        Waiting to discover 1 publishers ...
+        Waiting for data ...
+        Length:   464  Packets: 68081040  Packets/s(ave): 1134692  Mbps(ave):  4212.0  Lost:     0 (0.00%)
+        Finishing test...
+        Test ended.
+
+
+
+5. You can also launch *RTI Perftest* with your customType using DynamicData.
+
+.. code-block:: bash
+
+        ~/rtiperftest$ ./bin/x64Linux3gcc5.4.0/release/perftest_cpp -pub -executionTime 60 -noprint -dynamicData
+        RTI Perftest 3.0.0 (RTI Connext DDS 6.0.0)
+
+        Custom Type provided: 'Test'
+
+        Mode: THROUGHPUT TEST
+            (Use "-latencyTest" for Latency Mode)
+
+        Perftest Configuration:
+            Reliability: Reliable
+            Keyed: No
+            Publisher ID: 0
+            Latency count: 1 latency sample every 10000 samples
+            Data Size: 100
+            Batching: 8192 Bytes (Use "-batchSize 0" to disable batching)
+            Publication Rate: Unlimited (Not set)
+            Execution time: 60 seconds
+            Receive using: Listeners
+            Domain: 1
+            Dynamic Data: Yes
+            Asynchronous Publishing: No
+            XML File: perftest_qos_profiles.xml
+
+        Transport Configuration:
+            Kind: UDPv4 & SHMEM (taken from QoS XML file)
+            Use Multicast: False
+
+        Waiting to discover 1 subscribers ...
+        Waiting for subscribers announcement ...
+        Sending 4050 initialization pings ...
+        Publishing data ...
+        Setting timeout to 60 seconds
+        Length:   464  Latency: Ave    158 us  Std  166.5 us  Min     71 us  Max    678 us  50%    105 us  90%    169 us  99%    678 us  99.99%    678 us  99.9999%    678 us
+        Finishing test due to timer...
+        Test ended.
+
+
+
+
+.. code-block:: bash
+
+        ~/rtiperftest$ ./bin/x64Linux3gcc5.4.0/release/perftest_cpp -sub -noprint -dynamicData
+        RTI Perftest 3.0.0 (RTI Connext DDS 6.0.0)
+
+        Custom Type provided: 'Test'
+
+        Perftest Configuration:
+            Reliability: Reliable
+            Keyed: No
+            Subscriber ID: 0
+            Receive using: Listeners
+            Domain: 1
+            Dynamic Data: Yes
+            XML File: perftest_qos_profiles.xml
+
+        Transport Configuration:
+            Kind: UDPv4 & SHMEM (taken from QoS XML file)
+            Use Multicast: False
+
+        Waiting to discover 1 publishers ...
+        Waiting for data ...
+        Length:   464  Packets:  8146078  Packets/s(ave):  135770  Mbps(ave):   504.0  Lost:     0 (0.00%)
+        Finishing test...
+        Test ended.
```

Comment:
So, I would probably omit the DynamicData example and use a FlatData/ZeroCopy example instead.  I've not heard that customers are interested in the performance of DynamicData on their datatypes....not even sure what their use case would be myself.  But certainly, I have seen interest in FlatData (and mainly ZeroCopy which requires FlatData).